lib/kits/core-ui/page-decorations.styl

```diff
@@ -26,6 +26,67 @@
 
     .sgn-pagedecoration-hidden
         display none
+    
+    
+    .sgn-pagedecoration-hotspot
```

Comment:
Could you also unify the CSS with hotspots? Essentially extending hotspots to support links and videos.